### ISSUE TITLE
Only runs once and scrapes ~12 profiles

### ISSUE THREAD

**Issue by liamrosen** [gcwelborn/tinder-scraper#11]
I'm finding that upon running the scraper, it runs one time, gathers 12 profiles immediately, and then doesn't do anything else. 

Screenshot:
https://i.imgur.com/IFUM9bT.png
​
The readme claims that it will continue to run, but no more messages are displayed in Terminal, and the CSV doesn't get appended to.

What am I missing?

**Comment by timothe** [comment]
Thanks @TimBroddin for the fix :)

**Comment by liamrosen** [comment]
Thank you, Tim! But one issue now: the application seems to scrape the same profiles over and over in a never-ending loop. Was the idea that it is supposed to swipe left on a profile after scraping to make sure it doesn't come up again? If not, would this be trivial to add?

**Comment by timothe** [comment]
I'm pretty sure the profiles are different. Some are similar yes, but I achieve to have a varied set

**Comment by liamrosen** [comment]
OK, with some more sleuthing I figured a few things out:

1. If someone has liked your test profile, they'll appear over and over in the scrape. One profile appeared 3015 times out of 53791 profiles scraped because it was in my like queue.

2. After a certain point, even profiles not in your like queue will appear over and over. Tinder doesn't know where to end, so once it's scraped through all the unique profiles in the area, it will repeat. Here's a screenshot from the 53791 profile dataset with Conditional Formatting > Highlight Cells Rules > Duplicates selected in excel: https://i.imgur.com/qy51R9v.png

After selecting bios and removing duplicates, it turned out that only 7623 of the scraped profiles were unique out of my dataset of 53971.

tl;dr the scraper does work, as long as you are willing to do some work in excel to remove duplicates. An option to simply left swipe each profile as it's scraped would make sure that it knew where to stop scraping and prevent the duplicates issue.
